module/entities/helper.js

```diff
@@ -0,0 +1,208 @@
+
+                let projectionObject = {};
+
+                if (fields != "all") {
+                    fields.forEach(element => {
+                        projectionObject[element] = 1;
+                    });
+                }
+
+                let observationDocuments = await database.models.observations
+                    .find(queryObject, projectionObject)
+                    .lean();
+
+                return resolve(observationDocuments);
+            } catch (error) {
+                return reject(error);
+            }
+        });
+    }
+
+
+
+     /**
+   * Implement find query for entity
+   * @method
+   * @name entityDocuments
+   * @param {Object} [findQuery = "all"] - filter query object if not provide 
+   * it will load all the document.
+   * @param {Array} [fields = "all"] - All the projected field. If not provided
+   * returns all the field
+   * @param {Number} [limitingValue = ""] - total data to limit.
+   * @param {Number} [skippingValue = ""] - total data to skip.
+   * @returns {Array} - returns an array of entities data.
+   */
+
+     static entityDocuments(findQuery = "all", fields = "all", limitingValue = "", skippingValue = "",sortedData = "") {
+        return new Promise(async (resolve, reject) => {
+            try {
+                let queryObject = {};
+
+                if (findQuery != "all") {
+                    queryObject = findQuery;
+                }
+
+                let projectionObject = {};
+
+                if (fields != "all") {
+                    fields.forEach(element => {
+                        projectionObject[element] = 1;
+                    });
+                }
+
+                let entitiesDocuments;
+
+                if( sortedData !== "" ) {
+                entitiesDocuments = await database.models.entities
+                    .find(queryObject, projectionObject)
+                    .sort(sortedData)
+                    .limit(limitingValue)
+                    .skip(skippingValue)
+                    .lean();
+                } else {
+                    entitiesDocuments = await database.models.entities
+                    .find(queryObject, projectionObject)
+                    .limit(limitingValue)
+                    .skip(skippingValue)
+                    .lean();
+                }
+                
+                return resolve(entitiesDocuments);
+            } catch (error) {
+                return reject({
+                    status: error.status || httpStatusCode.internal_server_error.status,
+                    message: error.message || httpStatusCode.internal_server_error.message,
+                    errorObject: error
+                });
+            }
+        });
+    }
+
+
+       /**
+      * observation details.
+      * @method
+      * @name details
+      * @param  {String} observationId observation id.
+      * @returns {details} observation details.
+     */
+
+   static details(userToken, observationId) {
```

Comment:
@Prajwal17Tunerlabs observation details you need to add to observation helper not here

Reply:
added observation details in observation helper